src/components/Settings/shared/_variables.scss

```
@@ -59,6 +59,20 @@ $border-strong:  #555;   // hover-emphasized borders
 $font-title:   14px;
 $font-body:    13px;
 $font-caption: 12px;
+// The model cards' chips (language, tier, Recommended / Auto badges) share one
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)